src/components/Projects/EPCScopeOfWorkPage.jsx

```diff
@@ -0,0 +1,103 @@
+import { useState } from "react";
+import { AgGridColumn, AgGridReact } from 'ag-grid-react';
+import "ag-grid-enterprise";
+import Button from "../Form/Button";
```

Comment:
Group external libraries and custom file imports separately separated by a line

Reply:
Remove 'Page' from the file name. Page should be added to only components that have routes. And they have to be under pages folder